app/Services/Azure/MediaService.php

```diff
@@ -0,0 +1,211 @@
+    {
+        $payload = [
+            'properties' => [
+                'input' => [
+                    'assetName' => $assetInput,
+                    '@odata.type' => '#Microsoft.Media.JobInputAsset'
+                ],
+                'outputs' => [
+                    [
+                        'assetName' => $assetOutput,
+                        '@odata.type' => '#Microsoft.Media.JobOutputAsset'
+                    ]
+                ],
+                'priority' => 'Normal'
+            ]
+        ];
+
+        $uri = sprintf(
+            '/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Media/mediaServices/%s/transforms/%s/jobs/%s?api-version=%s',
+            config('streaming.azure.subscription_id'),
+            config('streaming.azure.resource_group'),
+            config('streaming.azure.account_name'),
+            $transformSlug,
+            $transformSlug . '_Job_' . date('Ymd-His'),
+            $this->apiVersion
+        );
+
+        $response = $this->client->put($uri, [
+            'headers' => ['Authorization' => 'Bearer ' . $this->apiKey],
+            'json' => $payload
+        ]);
+
+        return json_decode($response->getBody(), true);
+    }
+
+    public function getJobByName(string $jobName, $transformSlug = 'encodebolado')
+    {
+        $uri = sprintf(
+            '/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Media/mediaServices/%s/transforms/%s/jobs/%s?api-version=%s',
+            config('streaming.azure.subscription_id'),
+            config('streaming.azure.resource_group'),
+            config('streaming.azure.account_name'),
+            $transformSlug,
+            $jobName,
+            $this->apiVersion
+        );
+
+        $response = $this->client->get($uri, [
+            'headers' => ['Authorization' => 'Bearer ' . $this->apiKey]
+        ]);
+
+        return json_decode($response->getBody(), true);
+    }
+
+    public function getStreamingLocator(string $streamingLocatorName)
+    {
+        $uri = sprintf(
+            '/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Media/mediaServices/%s/streamingLocators/%s?api-version=%s',
+            config('streaming.azure.subscription_id'),
+            config('streaming.azure.resource_group'),
+            config('streaming.azure.account_name'),
+            $streamingLocatorName,
+            $this->apiVersion
+        );
+
+        $response = $this->client->get($uri, [
+            'headers' => ['Authorization' => 'Bearer ' . $this->apiKey]
+        ]);
+
+        return json_decode($response->getBody(), true);
+    }
+
+    public function createStreamingLocator(string $streamingLocatorName, string $assetName, string $streamingPolicyName = 'Predefined_ClearStreamingOnly')
+    {
+        $payload = [
+            'properties' => [
+                'assetName' => $assetName,
+                'endTime' => Carbon::now()->addYears(100)->toISOString(),
```

Comment:
100 anos e devera exegarado.

Reply:
Reclama com o tio bill pq eu to seguindo o tempo que eles estipularam.

Reply:
KKKKKKKKKKKKKKKKKKKKKKKKKKK como diria o ditado, Tio Bill é foda